AGAMI/Sources/Presentation/View/SignInView.swift

```diff
@@ -14,21 +14,37 @@ struct SignInView: View {
     @State private var viewModel: SignInViewModel = SignInViewModel()
     
     var body: some View {
-        VStack {
-            Spacer()
+        ZStack {
+            Image(.signInBackground)
+                .resizable()
+                .edgesIgnoringSafeArea(.all)
             
-            SignInWithAppleButton(.continue) { request in
-                viewModel.signInRequest(request: request)
-            } onCompletion: { result in
-                switch result {
-                case .success(let authorization):
-                    viewModel.handleSuccessfulLogin(with: authorization)
-                case .failure(let error):
-                    viewModel.handleLoginError(with: error)
+            VStack(spacing: 0) {
+                Spacer()
+                
+                Image(.signInLogo)
+                    .padding(EdgeInsets(top: 0, leading: 20, bottom: 24, trailing: 20))
```

Comment:
```suggestion
                Image(.signInLogo)
                    .resizable()
                    .scaleToFit()
                    .padding(EdgeInsets(top: 0, leading: 88, bottom: 24, trailing: 88))
```

resizable을 줘야 정확히 이미지 사이즈가 패딩에 따라 조정될거에요!
resizable이 없다면 leading, trailing을 준 게 적용이 안될 겁니다. 그리고 피그마에선 88로 나와있네요!